### PR TITLE
lci: add page

### PR DESCRIPTION
**Pull request by nath1as**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **v0.10.5**

### CODE REVIEW

File: pages/linux/lci.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# lci
+
+> LOLCODE interpreter written in C.
+> More information: <http://www.lolcode.org>.
+
+- Run a file written in LOLCODE:
+
+`lci {{path/to/file}}`
+
+- Display help:
```
Comment (kbdharun) [inline pages/linux/lci.md:10]: Can you move help command to last?

File: pages/linux/lci.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# lci
+
+> LOLCODE interpreter written in C.
+> More information: <http://www.lolcode.org>.
```
Comment (kbdharun) [inline pages/linux/lci.md:4]:
```suggestion
> More information: <https://github.com/justinmeza/lci>.
```
Their site seems to not work properly, it is better to point towards their GitHub I guess.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/lci.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# lci
+
+> LOLCODE interpreter written in C.
+> More information: <https://github.com/justinmeza/lci>.
+
+- Execute a file written in LOLCODE:
```
Comment (navarroaxel) [inline pages/linux/lci.md:6]: Based on `node` page
```suggestion
- Run a LOLCODE file:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**